.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,81 @@
+name: Bug report
+description: Report incorrect behavior in the Modin library
+title: 'BUG: '
+labels: ['bug 🦗', 'Triage 🩹']
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Modin version checks
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the latest version of Modin.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the main branch of Modin.
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
+        provide a minimal, copy-pastable example.
+      placeholder: >
+        import modin.pandas as pd
+
+        df = pd.DataFrame(range(5))
+
+        ...
+      render: python
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Issue Description
+      description: >
+        Please provide a description of the issue shown in the reproducible example.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: >
+        Please describe or show a code example of the expected behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Error Logs
+      description: >
+        Please paste the output of any relevant error logs.
+      value: >
+        <details>
+
+
+        Replace this line with the error backtrace (if applicable).
+
+
+        </details>
+  - type: textarea
+    id: version
+    attributes:
+      label: Installed Versions
```

Comment:
It might be helpful to clarify that `show_versions` gives machine information + versions for installed packages.

Are there any scenarios where a user would file a bug report where they're unable to import modin and consequently unable to call `show_versions`? We should describe what information we want from them if that's the case.

Reply:
Something would have to be terribly wrong if that were to happen, but yeah, we would probably want them to report other information if possible. I can modify the template to reflect that.

Reply:
including a stacktrace should be useful in such case

Reply:
On second thought, I think it might be simpler to just leave it as is. If `pd.show_versions()` doesn't work then we can handle those cases specifically.